src/spatial/reaction_rates.jl

```diff
@@ -52,20 +52,13 @@ end
 
 update rates of all reactions in rxs at site
 """
-function update_rx_rates!(rx_rates::RxRates{F, M}, rxs, u,
+function update_rx_rates!(rx_rates::RxRates{F, M}, rxs, integrator,
                           site) where {F, M <: MassActionJump}
```

Comment:
```suggestion
                          site) where {F, M <: AbstractMassActionJump}
```
otherwise this won't apply to `SpatialMassActionJump`s will it?

Reply:
Yes, thank you.